### ISSUE TITLE
[Adaptive Cards - Schema Explorer] - Lang attribute is not defined for the 'هذه كتلة نص' and 'זהו בלוק טקסט' text present in the 'Adaptive Card' section on the page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9304]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Tool: Developer Tool

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Schema Explorer' present on the header section of the page using tab key and activate using enter key.
3. Now navigate to the 'هذه كتلة نص' and 'זהו בלוק טקסט' text present in the 'Adaptive Card' section on the page using tab key.
4. Press F12 key to open developer tool and inspect the code.
5. Verify that, lang attribute is defined for the 'هذه كتلة نص' and 'זהו בלוק טקסט' text present in the 'Adaptive Card' section on the page or not.

**Actual Result:**
Lang attribute is not defined for the 'هذه كتلة نص' and 'זהו בלוק טקסט' text present in the 'Adaptive Card' section on the page.

**Reproducibility:**
Same issue replicates for the different language content such as 'Lorem ipsum dolor sit amet,...' etc. present in the 'Adaptive Card' section on the 'TextBlock' and 'RichTextBlock' page. **Refer Attachment:** Reproducibility_Lang

**Expected Result:**
Correct Lang attribute should be defined for the 'هذه كتلة نص' and 'זהו בלוק טקסט' text present in the 'Adaptive Card' section on the page. The lang attribute should have complete language code with country/region dialect. For example, if language is 'Arabic' then the attribute should be 'ar' and if language is 'Hebrew' then the attribute should be 'he'.

**User Experience:**
Visually impaired users who rely on screen readers or other technologies that convert text into synthetic speech will be impacted if lang attribute is not defined for the 'هذه كتلة نص' and 'זהו בלוק טקסט' text present in the 'Adaptive Card' section on the page which is present in different language due to which AT will not be able to correctly present content written in multiple languages. So, it will be difficult for the user agents and assistive technologies to present content according to the presentation and pronunciation rules for that language.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/language-of-parts.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/9788b010-06e8-4b0e-9a8a-e022e3f68510

![Image](https://github.com/user-attachments/assets/54c53c81-d7bd-4e5b-91de-ae53029d1783)

**Reproducibility_Lang**

![Image](https://github.com/user-attachments/assets/e0defce8-fd5a-477a-bafa-06a99bd7bffa)

### Card JSON

```JSON
NA
```
